crates/ty_python_semantic/resources/mdtest/import/star.md

```diff
@@ -182,16 +182,19 @@ class ContextManagerThatMightNotRunToCompletion:
 with ContextManagerThatMightNotRunToCompletion() as L:
     U = ...
 
-match 42:
+def get_object() -> object:
+    pass
+
+match get_object():
     case {"something": M}:
         ...
     case [*N]:
         ...
     case [O]:
         ...
-    case P | Q:  # error: [invalid-syntax] "name capture `P` makes remaining patterns unreachable"
+    case I(foo=R):
         ...
-    case object(foo=R):
+    case P | Q:
```

Comment:
I changed the order here (and matched on `I(…)` instead of `object(…)`). The test would fail otherwise because we would detect the `P | Q` pattern to be always reachable (and the ones behind it to not be reachable). I don't think this was the original intention of this test (or the assertion below should have a TODO).